### PR TITLE
Add github project slug to API catalogue entities

### PR DESCRIPTION
**Pull request by garryod**

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ metadata:
   name: blueapiControl
   title: Athena BlueAPI Control
   description: REST API for getting plans/devices from the worker (and running tasks)
+  annotations:
+    github.com/project-slug: DiamondLightSouce/blueapi
```
Comment (RAYemelyanova) [inline catalog-info.yaml:26]: looks good but I'm unsure of it's purpose. Could I ask what this does?

**Review by RAYemelyanova — APPROVED** [review]

File: catalog-info.yaml (hunk 1 above)
Comment (garryod) [reply catalog-info.yaml:26]: It's used by a few of the plugins, the first one that pops to mind the GitHub actions one which will allow users to view the CI history of the project from the portal

Comment (keithralphs) [reply catalog-info.yaml:26]: Haven't we already got this on line 8, except it seems to have been misspelled as DiamondLightSouce? Why do we need it in the API section too, don't just need to correct the spelling on line 8?

**Review by keithralphs — CHANGES_REQUESTED** [review]

File: catalog-info.yaml (hunk 1 above)
Comment (garryod) [reply catalog-info.yaml:26]: Aha, good catch, I had completely missed that typo :sweat_smile: For needing it in multiple sections - as far as the portal is concerned these entities are entirely independent, hence if we want any of the goodies that come with linking to the repo (e.g. the aforementioned CI history) on the APIs then we need to include it in their definitions too

Comment (keithralphs) [reply catalog-info.yaml]: Ok fair enough then

**Review by keithralphs — APPROVED** [review]

**Pull request merged**